toonz/sources/translations/german/toonzlib.ts

```diff
@@ -0,0 +1,610 @@
+    </message>
+    <message>
+        <source>Can&apos;t save an empty level</source>
+        <translation>Kann keine leere Ebene speichern</translation>
+    </message>
+    <message>
+        <source>Unrecognized file type :</source>
+        <translation>Unbekannter Dateityp :</translation>
+    </message>
+    <message>
+        <source>Can&apos;t save a %1 level to this file type : %2</source>
+        <translation>Kann keine %1 Ebene zu diesem Dateityp speichen : %2</translation>
+    </message>
+    <message>
+        <source>Exception writing %1</source>
+        <translation>Ausnahme beim schreiben von %1</translation>
+    </message>
+    <message>
+        <source>frame index (%1) must be a number</source>
+        <translation>Frame-Index (%1) muss eine Nummer sein</translation>
+    </message>
+    <message>
+        <source>frame index (%1) is out of range (0-%2)</source>
+        <translation>Frame-Index (%1) ist außerhalb des Bereichs (0-%2)</translation>
+    </message>
+    <message>
+        <source>second argument (%1) is not an image</source>
+        <translation>zweites Argument (%1) ist kein Bild</translation>
+    </message>
+    <message>
+        <source>can not insert a %1 image into a level</source>
+        <translation>kann nicht ein %1 Bild in eine Ebene einfügen</translation>
+    </message>
+    <message>
+        <source>can not insert a %1 image to a %2 level</source>
+        <translation>kann nicht ein %1 Bild in eine %2 Ebene einfügen</translation>
+    </message>
+</context>
+<context>
+    <name>TScriptBinding::OutlineVectorizer</name>
+    <message>
+        <source>Can&apos;t vectorize a %1 level</source>
+        <translation>Kann keine %1 Ebene vektorisieren</translation>
+    </message>
+    <message>
+        <source>Can&apos;t vectorize a level with no frames</source>
+        <translation>Kann keine Ebene ohne Frames vektorisieren</translation>
+    </message>
+    <message>
+        <source>Can&apos;t vectorize a %1 image</source>
+        <translation>Kann kein %1 Bild vektorisieren</translation>
+    </message>
+    <message>
+        <source>Bad argument (%1): should be an Image or a Level</source>
+        <translation>Inkorrektes Argument (%1): Es sollte ein Bild oder eine Ebene sein</translation>
+    </message>
+    <message>
+        <source>Invalid color : </source>
+        <translation>Ungültige Farbe : </translation>
+    </message>
+</context>
+<context>
+    <name>TScriptBinding::Rasterizer</name>
+    <message>
+        <source>Expected a vector image: %1</source>
+        <translation>Es ist kein Vektor-Bild: %1</translation>
+    </message>
+    <message>
+        <source>Expected a vector level: %1</source>
+        <translation>Es ist keine Vektor-Ebene: %1</translation>
+    </message>
+    <message>
+        <source>Argument must be a vector level or image : </source>
+        <translation>Argument muss eine Vektor-Ebene oder Bild sein : </translation>
```

Comment:
Two separate things are meant here I think, at least that's what I get from the English and Japanese versions